Core/Steps/PipelineSteps/BranchFromMasterStep.cs

```diff
@@ -66,7 +66,7 @@ public void Execute (string? commitHash, bool startReleasePhase, bool pauseForCo
     _semanticVersionedGitRepository.TryGetCurrentVersion(out var currentVersion, "master");
     _log.Debug("The current found version is '{CurrentVersion}'.", currentVersion);
     
-    var nextVersion = FindNextPatch(currentVersion ?? throw new InvalidOperationException("No current version exists."));
+    var nextVersion = FindNextPatch(currentVersion ?? throw new UserInteractionException("No current version exists."));
```

Comment:
Can you make this message a bit more meaningful / more context? Why would a current version not exist. Do have have typical cases that we should tell the user here?